La'Tonia/README.md

```diff
@@ -1,32 +1,44 @@
-# <div style="text-align: center">**JUSTICE THROUGH CODE**</div>
-### *confession by a code master hopeful*
+# <div align="center">**JUSTICE THROUGH CODE**</div>
```

Comment:
We'll get a lot of practice with this when we get into html and css files - I haven't worked too much with them but this is amazing practice you're getting into!